pulseapi/entries/views.py

```diff
@@ -462,11 +462,11 @@ def post(self, request, *args, **kwargs):
                 )
 
                 # create entry/creator intermediaries
-                if creator_data is not None:
+                if saved_entry.posted_by_creator:
+                    creator_data.append(user.name)
```

Comment:
based on the slack convo, my guess is that atm we don't care about the order in which we add the submitter as a creator?

Reply:
yeah. Although I did just file #223 so we can revisit after MVP